src/copy-paste.js

```diff
@@ -330,7 +330,8 @@ define([
             $(document).on('cut copy paste keydown', function (e) {
                 if (e.type === 'cut' ||
                     e.metaKey && String.fromCharCode(e.keyCode) === 'X') {
-                    onCut(opts);
+                    // Disable cut until undo feature is implemented
+                    if (false) { onCut(opts); }
```

Comment:
Used a never-true conditional because otherwise the `onCut` function caused a jshint warning: 'onCut' is defined but never used.